### DISCUSSION TITLE
Postal Initialize results in "unknown shorthand flag: 'p' in -p See 'docker --help'."

### DISCUSSION THREAD

**Discussion by nghng** [postalserver/postal#3250] [Help with using Postal] [answered]
I have tried to install docker-compose and update my docker, but the same error persists. I saw a post having the same error but with podman. However, the answer was to install docker with the latest version.
Any help would be greatly appreciated!

Thanks a lot in advance!

![image](https://github.com/user-attachments/assets/649f00af-2119-426e-97eb-af42d72e5e1c)

**Comment by nghng** [comment]
![image](https://github.com/user-attachments/assets/0e01451a-ddda-4b78-bfeb-d9bd73e05145)
Here's mine docker version, I have removed the version tag in the /opt/postal/install/docker-compose.yml. Yet the error still appears

**Reply by willpower232** [reply, marked as answer]
I presume you're lacking the compose plugin, it may also be slightly out of date since I'm up to 27 but docker aren't very good at version numbers so that might not mean anything.

**Reply by nghng** [reply]
Thanks for answering, it was indeed something wrong with my docker-compose plugin
